src/client/src/firebaseConfig.ts

```diff
@@ -0,0 +1,10 @@
+export const firebaseConfig = {
```

Comment:
https://stackoverflow.com/questions/37482366/is-it-safe-to-expose-firebase-apikey-to-the-public#:~:text=It%20is%20not%20a%20security,uses%20Firebase%20as%20its%20backend.